### PR TITLE
Deprecate old AMIs (keep 180)

### PR DESCRIPTION
**Pull request by peterdeme**
## Description of the change

Deprecate old AMIs and keep only 80.

Plus modernize the build process, we used an outdated package.

## Type of change

- [ ] Bug fix (non-breaking change that fixes an issue);
- [x] New feature (non-breaking change that adds functionality);
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected);
- [ ] Documentation (a documentation or example fix not affecting the infrastructure managed by this module);

### Development

- [ ] All necessary variables have been defined, with defaults if applicable;
- [ ] The HCL code is formatted;
- [ ] An AMI has been created in some AWS account, and the AMI is working as expected;

### Code review

- [ ] This pull request has a descriptive title and information useful to a reviewer. There may be a screenshot or screencast attached;
- [ ] This pull request is no longer marked as "draft";
- [ ] Reviewers have been assigned;
- [ ] Changes have been reviewed by at least one other engineer;

### CODE REVIEW

File: .github/workflows/build_aws_scheduled.yml [hunk 1]
```diff
@@ -41,17 +41,16 @@ jobs:
           role-to-assume: ${{ secrets.AWS_ROLE_ARN }}
           role-duration-seconds: 3600
 
-      - name: Validate the Packer template
-        uses: hashicorp/packer-github-actions@master
+      - name: Setup packer
+        uses: hashicorp/setup-packer@main
```
Comment (peterdeme) [inline .github/workflows/build_aws_scheduled.yml:45]: This has nothing to do with this task but `hashicorp/packer-github-actions` got deprecated so I needed to update it everywhere.

**Comment by peterdeme** [comment]
@michieldewilde @adamconnelly this is ready to review, AWS has bumped our quota.

**Comment by peterdeme** [comment]
@michieldewilde yeah, apparently the DescribeImageAttribute API can do that

https://docs.aws.amazon.com/AWSEC2/latest/APIReference/API_DescribeImageAttribute.html

**Comment by peterdeme** [comment]
@michieldewilde I mean what blocks us from having to automate that? Suggestion:

- mark AMI as deprecated after 40 weeks

Then:
- look for AMIs that are older than 40 weeks and are NOT used by anyone (last launch time = null) OR last launch time is more than 6 months, and delete them

This way, we'd get rid of a bunch of AMIs, but we would also implicitly tell the customers to update their images ASAP by deprecating them. I assume you get a notification from AWS if you're using a deprecated AMI or something.

**Review by michieldewilde — DISMISSED** [review]

**Pull request merged**